.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 4
```

Comment:
:speech_balloon: I'm more of a 2-space person these days, but less of a concern with Markdown I guess